proof/refine/ARM/Tcb_R.thy

```diff
@@ -3614,42 +3713,54 @@ lemma decodeSetSpace_tc_slot[wp]:
   apply simp
   done
 
-lemma decode_tcb_conf_corres:
+lemma decodeTCBConfigure_corres:
```

Comment:
We probably want to defer updating the names of existing lemmas (those that are also in master) for now, just to save the extra work it might generate while merging. We’ll clean them up altogether at some point.

But we can be consistent and proper when we name the rules for the functions we are adding newly for MCS.

Reply:
Well, the merge is going to be horrific anyway, right? My vote is to change whatever we like now and deal with an ever so slightly worse merge later. I've found the cost of not following the conventions is actually quite high; if you think there should be a lemma called `some_lemma`, but it's actually called `some_Lemma_haha` then it's quite annoying

Reply:
Tracking issue: [VER-1386](https://sel4.atlassian.net/browse/VER-1386)